packages/appstore/src/components/loader/cfd-app-skeleton/cfdAppskeleton.scss

```diff
@@ -0,0 +1,31 @@
+@import '../skeletonLoader.scss';
+
+.cfd-app-skeleton-loader {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+
+    &__header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 8px;
+
+        &__icon {
+            height: 38px;
+            border-radius: 8px;
+            width: 57px;
+        }
+        &__title {
+            height: 24px;
+            border-radius: 8px;
+            width: 106px;
+        }
+    }
+
+    &__body {
+        height: 24px;
+        border-radius: 8px;
+        width: 422px;
+    }
+}
```

Comment:
just a nit there's a lot of `border-radius: 8px;` you can define it at the top of the file as `$border_radius: 8px`. 
If there's any changes you can just change in one place.